app/src/main/res/values/styles.xml

```diff
@@ -36,14 +36,38 @@
 		<item name="android:background">@drawable/keyboard_button_selector</item>
 	</style>
 
+	<style name="ListItemIcon">
+		<item name="android:layout_marginLeft">@dimen/list_item_left_right_padding</item>
+		<item name="android:layout_marginStart" tools:ignore="NewApi">@dimen/list_item_left_right_padding</item>
+		<item name="android:layout_centerVertical">true</item>
+		<item name="android:layout_alignParentLeft">true</item>
+		<item name="android:layout_alignParentStart" tools:ignore="NewApi">true</item>
+	</style>
+
 	<style name="ListItemFirstLineText">
+		<item name="android:layout_alignParentTop">true</item>
 		<item name="android:textColor">?android:textColorPrimary</item>
-		<item name="android:textSize">16sp</item>
+		<item name="android:textSize">@dimen/list_item_left_right_padding</item>
```

Comment:
This is a bit weird...  Did you mean to use padding for text size? Note that you should use sp instead of dp for font sizes because it can scale with the user's os-level font size preferences.